nthprime/spec/nthfibonacci_spec.rb

```diff
@@ -0,0 +1,27 @@
+require 'nthfibonacci'
+require 'minitest/autorun'
+
+class NthFibonacciTest < Minitest::Test
```

Comment:
Great handful of tests on this project. It's awesome to see so many angles taken into consideration.